CodeModel.Tests/DataFactories/CodeFactory.cs

```diff
@@ -66,6 +66,7 @@ internal bool IsAvailable(DateRange timeSpan)
             }"
             };
         }
```

Comment:
@Luburic I've formatted this file by mistake. Should this be reverted, or it's fine to say?

Reply:
It's fine.